.github/workflows/publish-pypi.yaml

```
@@ -0,0 +1,27 @@
+name: Publish-PyPI
+
+on:
+  push:
+    tags: v[0-9]+.[0-9]+.[0-9]+*
+
+jobs:
+  upload-pypi:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```
I think `v3` causes deprecation warnings in actions.